src/components/TableHeader.tsx

```diff
@@ -140,13 +167,13 @@ export const TableHeader = ({
       )}
       {/* Reset to default settings button.
       TODO: This should be obfuscated final build */}
-      <button onClick={handleForceFirstTimeSetup}
+      {/* <button onClick={handleForceFirstTimeSetup}
```

Comment:
do we plan on using this stuff later? just wondering if theres a good reason to keep it around

Reply:
no i just have commitment issues.